frontend/app/models/recurrence-match-days.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "Monday" }),
```

Comment:
@nogates please note that in Ruby's date of the week `0 == 7 == "Sunday"`

I am not sure if this code is wired to the API call